### ISSUE TITLE
Strange frame rate capping on my Tablet

### ISSUE THREAD

**Issue by trsh** [4ian/GDevelop#2092]
## Describe the bug

When I export APK and run on my tablet FPS is always capped to 30 FPS (even empty scene). Also when I run over network it lags like 30 fps unless I enable this event:

![ffff](https://user-images.githubusercontent.com/3727523/99156542-3c309c00-26ca-11eb-872e-76dc722a5291.png)

then its capped to 56 fps no matter the settings. This is default platformer I am experimenting with

[Test22.zip](https://github.com/4ian/GDevelop/files/5541428/Test22.zip)

, but same is with empty scene.

## To Reproduce
1. Take this project and run over network on similar Tablet
2. See how FPS changes (capped 30 and 56) when this

![ffff](https://user-images.githubusercontent.com/3727523/99156550-4f436c00-26ca-11eb-99cf-574d4303878a.png)

is enabled or disabled. Very weird.

3. Try to export APK using builtin exporter (with line above enabled to see fps) and install on similar Tablet
4. Observe FPS capped to 30

## Other details
* Tablet is Lenovo TAB 2 A10-70F
* Gdevelop 5.0.0-beta101 desktop on windows 10

**Comment by trsh** [comment]
Tested in my phone BV9100, over network, FPS jumps from 31-61 constantly.

**Comment by trsh** [comment]
Also on my Phone BV9100 Blackview, over network, FPS jumps from exactly from 31-61 fps

**Comment by 4ian** [comment]
By itself, these framerates are expected.
A screen has a refresh rate, which is usually 60hz, i.e: 60 images per second. In other words, an image every 16.666 milliseconds.
If your game logic + rendering is done under 16ms, then the image will be sent by the GPU to the display, then the screen will show it at the next refresh. The game will be waiting at this point to start rendering the next image (depends on your browser, but that's likely what is happening).

If your game is not rendering anything in under 16ms - let's say that the total time is 23ms -, then the display will keep showing the existing image on screen. Then when your game finishes rendering the image, it will submit it to the display which will show it at its next "refresh"..., which will be 16.666+16.666=33.333 milliseconds after the last image.
In other words, the number of frame per seconds will be 1/33.333 = 30fps.
 
(in practice they are slight delays there and there, so the FPS shown on screen is 31/61fps).

*This being said*, it's weird that just setting a text would dramatically impact the time to render so that it goes from under 16ms to above. It might be that rendering the text is very costly on your tablet - which is possible.

How do you measure the number of FPS when the text is not set to `1/TimeDelta()`?

**Comment by trsh** [comment]
@4ian the true mystery here is why APK run on my tablet on constant capped 30 FPS (this should be investigated), but trough network thing in browser it can do 58 fps. It doesn't make sense, unless the Build some-have enforces this limit. Maybe there is some bug in packing process?

> It might be that rendering the text is very costly on your tablet - which is possible.

Its the opposite. Whenever I ENABLE this counter the fps goes normal, when I remove it I can tell with my eye, the fps is back to 30 or worse. I know it doesn't make sense, but calling `TimeDelta()` in event some-have improves game performance :D
